.github/workflows/job_build_windows.yml

```diff
@@ -33,7 +37,7 @@ jobs:
     runs-on: ${{ inputs.runner }}
     env:
       CMAKE_BUILD_TYPE: ${{ inputs.build-type }}
-      CMAKE_GENERATOR: 'Ninja Multi-Config'
+      CMAKE_GENERATOR: 'Ninja'
```

Comment:
why?

Reply:
2024-10-15T12:06:24.7561511Z -- Build files have been written to: C:/actions-runner/_work/openvino/openvino/openvino_build/src/plugins/intel_gpu/thirdparty/onednn_gpu_build
2024-10-15T12:06:24.7570304Z [4387/9080] C:\Windows\system32\cmd.exe /C "cd /D C:\actions-runner\_work\openvino\openvino\openvino_build\src\plugins\intel_gpu\thirdparty\onednn_gpu_build && "C:\Program Files\CMake\bin\cmake.exe" --build . --config Debug && "C:\Program Files\CMake\bin\cmake.exe" -E touch C:/actions-runner/_work/openvino/openvino/openvino_build/src/plugins/intel_gpu/thirdparty/onednn_gpu_root/src/onednn_gpu_build-stamp/Debug/onednn_gpu_build-build"
2024-10-15T12:06:24.7654068Z FAILED: src/plugins/intel_gpu/thirdparty/onednn_gpu_root/src/onednn_gpu_build-stamp/Debug/onednn_gpu_build-build src/plugins/intel_gpu/thirdparty/onednn_gpu_install/lib/openvino_onednn_gpu.lib C:/actions-runner/_work/openvino/openvino/openvino_build/src/plugins/intel_gpu/thirdparty/onednn_gpu_root/src/onednn_gpu_build-stamp/Debug/onednn_gpu_build-build C:/actions-runner/_work/openvino/openvino/openvino_build/src/plugins/intel_gpu/thirdparty/onednn_gpu_install/lib/openvino_onednn_gpu.lib

Reply:
And we don't really use multiconfiguration in CI at all

Reply:
>  multiconfiguration in CI at all

But CI is used for validation, so we have validated this mode.